src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -1363,6 +1363,68 @@ public IActionResult SyncFromDate([FromBody] WalletSyncFromDateRequest request)
             return this.Ok();
         }
 
+        [Route("wallet-stats")]
+        [HttpGet]
+        public IActionResult WalletStats([FromQuery] WalletStatsRequest request)
+        {
+            Guard.NotNull(request, nameof(request));
+            WalletStatsModel model = new WalletStatsModel();
+            model.WalletName = request.WalletName;
+
+            // Checks the request is valid.
+            if (!this.ModelState.IsValid)
+            {
+                return ModelStateErrors.BuildErrorResponse(this.ModelState);
+            }
+
+            try
+            {
+                IEnumerable<UnspentOutputReference> spendableTransactions = this.walletManager.GetSpendableTransactionsInAccount(new WalletAccountReference(request.WalletName, request.AccountName), request.MinConfirmations);
+
+                model.TotalUtxoCount = spendableTransactions.Count();
+                model.UniqueTransactionCount = (from s in spendableTransactions
```

Comment:
Ah yes this was done differently in @maciejzaleski PR. So yeah we need to pull in master.

Reply:
Done.